docs/proposals/002-api-proposal/proposal.md

```diff
@@ -0,0 +1,313 @@
+
+# LLM Instance Gateway
```

Comment:
Please correct if I'm wrong here, can I say this is more like a LLM Instance Orchestrator? The current major use case is trying to provision LoRA instances to different Pods via some use cases?

Reply:
The gateway is only responsible to route incoming requests to the right (regarding fairness, availability, load...) BackendPool running the LoRA adapter. Loading the LoRA adapter is the responsibility of the workload author. Then, it's out of the scope of the gateway.  You can [refer to the PoC](https://github.com/kubernetes-sigs/llm-instance-gateway/tree/main/examples/poc) for further details.

Reply:
@Joffref is correct. This is not about provisioning of adapters, but we do define them in the UseCase as they are the initial focus of how we intend to specify sharing of compute capacity.

+1 to refering to the PoC or the demo recording for more info. Also if we feel explanations are insufficient please lmk!